### ISSUE TITLE
Error installing the systemd service on Fedora 30

### ISSUE THREAD

**Issue by serovar** [Jackett/Jackett#6098]
sudo ./install_service_systemd.sh
Failed to stop jackett.service: Unit jackett.service not loaded.
Unit jackett.service could not be found.
Created symlink /etc/systemd/system/multi-user.target.wants/jackett.service → /etc/systemd/system/jackett.service.
ERROR: Could not launch service. The installation might have failed.
Please open an issue on https://github.com/Jackett/Jackett/issues and paste following information:
Jackett directory: `/home/nonesim/Jackett`
Jackett user: `nonesim`

From the journalctl log:
`Oct 05 19:54:28 localhost-live systemd[1]: Started Jackett Daemon.
Oct 05 19:54:28 localhost-live systemd[3863]: jackett.service: Failed to execute command: Permission denied
Oct 05 19:54:28 localhost-live systemd[3863]: jackett.service: Failed at step EXEC spawning /home/nonesim/Jackett/jackett: Permission denied
Oct 05 19:54:28 localhost-live systemd[1]: jackett.service: Main process exited, code=exited, status=203/EXEC
Oct 05 19:54:28 localhost-live systemd[1]: jackett.service: Failed with result 'exit-code'.
Oct 05 19:54:34 localhost-live systemd[1]: jackett.service: Service RestartSec=5s expired, scheduling restart.
Oct 05 19:54:34 localhost-live systemd[1]: jackett.service: Scheduled restart job, restart counter is at 1.
Oct 05 19:54:34 localhost-live systemd[1]: Stopped Jackett Daemon.
`

I am using a fresh install of Fedora 30 with Jackett v0.11.768

**Comment by garfield69** [comment]
which installation package did you download?

**Comment by serovar** [comment]
Jackett.Binaries.LinuxAMDx64.tar.gz

The Jackett binary works, the problem is only with the systemd service installing script.

If I use the Jackett package in the official Fedora repos the systemd service works fine.

**Comment by garfield69** [comment]
you appear to be getting a "not authorised" permission error.
did you remember to change your terminal path to the folder that contains the jackett suite before issuing the systemd install_Service command?
if you did this, and got the permission error, then there is something awry with the permission attributes of the jackett folder that contains everything.
if you did not change the path to the jackett folder, then running the script from a different location might account for the permission error. so switch to the correct folder and try the script again.

**Comment by serovar** [comment]
I have the Jackett folder in my home folder, I did a `cd Jackett/` and then I ran the script.

**Comment by garfield69** [comment]
hmmm.
looking back at some of the open issues I came across #5533
it suggested the Jackett installation directory and/or files must be owned by whatever user Jackett is going to run as. 
check the script to see what the user is set as and then run
sudo chown -R <jackettuser> <JackettInstallDirectory>

**Comment by serovar** [comment]
I saw that issue, but I don't think this is the case here, since as you can see in my first post my Jackett user is the correct one (nonesim) and not UNKNOWN. I already checked the systemd script and the Jackett folder permissions, and they are both referring to user nonesim and group nonesim. I also ran `sudo chown -R nonesim /home/nonesim/Jackett/` and `sudo chown -R nonesim:nonesim /home/nonesim/Jackett/` just to be sure, but nothing changed.

**Comment by garfield69** [comment]
then I am sorry but I cannot help further.
I installed the AMDx64 package on my win10 virtualbox running an Ubuntu18 image, to ~/Downloads/Jackett, cd to that folder and ran the script without problems.
something on the Fedore system would appear to be blocking the permissions? 
I just don't know.

**Comment by serovar** [comment]
Thanks anyway for your help, I hope someone with more knowledge of the Fedora system than us could chime in.

**Comment by elpicoso** [comment]
Bumping, having the same issue on my pi

**Comment by garfield69** [comment]
@elpicoso  amdx64 is the wrong package for pi.
your best bet is the arm32, and following https://github.com/Jackett/Jackett#install-on-linux-armv7-or-above

**Comment by garfield69** [comment]
@serovar  wrote:
>If I install the Jackett package in the official Fedora repos the systemd service works fine.

can you provide a link to this package you say has a systemd install that works for you?
I want to compare the script with the one in the amd64 package and see if I can spot a difference.

**Comment by serovar** [comment]
https://negativo17.org/repos/multimedia/fedora-29/x86_64/jackett-0.11.761-1.fc29.noarch.rpm

I edited my comment in which I thought it was in the official repos.

**Comment by garfield69** [comment]
I took at look at that archive. i found the following in `\usr\lib\systemd\system\jackett.service`
```
[Unit]
Description=API Support for your favorite torrent trackers

[Service]
ExecStart=/usr/bin/mono --debug /usr/share/jackett/JackettConsole.exe --NoUpdates
User=jackett
Group=jackett
Restart=on-failure
RestartSec=5

[Unit]
After=network.target

[Install]
WantedBy=multi-user.target
```
if this is the systemd profile that that package is installing, then it would seem that that package is using a mono version of jackett.

what is the cpu processor in your fedora system? you need to look it up in wiki to check whether you are on the ARM or AMD family of processors.

**Comment by serovar** [comment]
Yes, it is using the mono version, unfortunately. My CPU is an Intel i7-2600, not an ARM processor.

**Comment by garfield69** [comment]
so it would appear that the fedora 30 does not support the AMD .net core libraries then.
and you are stuck waiting for someone to build an ARM version of the mono jackett suite using the latest jackett source.

what was the reason you wanted to upgrade to a more recent version of jackett anyway?

**Comment by garfield69** [comment]
mind you, you said that the amd binaries work and that you can start jackett manually correct?

**Comment by garfield69** [comment]
so all you need to do is update the `\usr\lib\systemd\system\jackett.service` file to startup the non-mono version.

**Comment by garfield69** [comment]
so you change the jackett.service to
```
WorkingDirectory=/home/nonesim/Jackett
ExecStart=/home/nonesim/Jackett/jackett --NoRestart
```
and see what happens?

**Comment by serovar** [comment]
This is already the content of jackett.service, unfortunately.

```
[Unit]
Description=Jackett Daemon
After=network.target

[Service]
SyslogIdentifier=jackett
Restart=always
RestartSec=5
Type=simple
User=nonesim
Group=nonesim
WorkingDirectory=/home/nonesim/Jackett
ExecStart=/home/nonesim/Jackett/jackett --NoRestart
TimeoutStopSec=20

[Install]
WantedBy=multi-user.target
```

**Comment by garfield69** [comment]
in the `\usr\lib\systemd\system` folder?

**Comment by garfield69** [comment]
do you think when you did the `sudo chown -R nonesim:nonesim /home/nonesim/Jackett/` that the last slash is significant? have you tried `sudo chown -R nonesim:nonesim /home/nonesim/Jackett` ?
other than that I've run out of ideas :-(

**Comment by garfield69** [comment]
cause the install script in the amd64 pack wants to create `jackett.service` in `/etc/systemd/system`
but your fedora expects it to be in `\usr\lib\systemd\system`

**Comment by serovar** [comment]
I tried moving the file there but it does not work. Thanks for your support, I'll stick with the mono version for now.

**Comment by Fr8Hauler** [comment]
Had the same problem, fixed in a few hours, trial and error. My solution was adding `/bin/bash` after `ExecStart=` 
Before: `ExecStart=/home/fr8haul/Jackett/jackett_launcher.sh`
After: `ExecStart=/bin/bash /home/fr8haul/Jackett/jackett_launcher.sh` 
to the `jackett.service` file
Also moved service file from `/etc/systemd/system` to `/usr/lib/systemd/system`

BTW, running Centos 8

**Comment by egadsthefuzz** [comment]
Had the same thing on fedora 31 and I suspect it would be the same on centos/rhel.
Line like this sorted it `ExecStart=/bin/bash /home/fr8haul/Jackett/jackett_launcher.sh`
Would it hurt to just add the /bin/bash globally as that is technically correct and systemd might break on other systems down the line

**Comment by ngosang** [comment]
Closed by #7614

**Comment by garfield69** [comment]
Jackett 0.13.611
